lib/postProcessing/include.js

```diff
@@ -186,28 +184,15 @@ includePP._fillIncludeTree = function(includeTree, request, callback) {
     var parts = related.as.split("~~");
     debug.include(related);
 
-    externalRequest({
+    rerouter.route({
       method: "GET",
       uri: related.url,
-      headers: request.safeHeaders
-    }, function(err, res, json) {
-      // console.log(err, json)
-      if (err || !json) {
-        return done(null);
-      }
-
-      try {
-        json = JSON.parse(json);
-      } catch(e) {
-        debug.include("!!", JSON.stringify(json));
-        json = null;
-      }
-
-      if (!json) return done(null, [ ]);
-
-      if (res.statusCode >= 400) {
-        debug.include("!!", JSON.stringify(json));
-        return done(json.errors);
+      headers: request.safeHeaders,
+      cookies: request.cookies
+    }, function(err, json) {
+      if (err) {
+        debug.include("!!", JSON.stringify(err));
```

Comment:
Worth having a namespaced debug for `rerouter`? The above might be difficult to identify when looking at debug output otherwise.